### PR TITLE
[SWA-129][FEAT] - Add Scroll chain support

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-129](https://linear.app/swaprhq/issue/SWA-129/add-scroll-chain)

# Description

* Update the pre-selected output currency for Gnosis, to be WETH

# Visual Evidence
image

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Connect your wallet
5) Connect to Scroll chain and try to check balances and/or do a swap. The token quotes should appear through Sushi v3

### CODE REVIEW

**Comment by Diogomartf** [comment]
image

on the screenshot Sushiswap appear twice, that's not supposed right?

File: src/components/NetworkSwitcher/NetworkSwitcher.preset.ts [hunk 1]
```diff
@@ -54,6 +55,12 @@ export const networkOptionsPreset: NetworkOptionsPreset[] = [
     logoSrc: ZkSyncEraLogo,
     color: '#1E69FF',
   },
+  {
+    chainId: ChainId.SCROLL_MAINNET,
+    name: 'Scroll',
+    logoSrc: ScrolLogo,
+    color: '#1E69FF',
```
Comment (Diogomartf) [inline src/components/NetworkSwitcher/NetworkSwitcher.preset.ts:62]: this should be scroll color.

I think this one works: `#ffdbb0`

**Comment by Diogomartf** [comment]
Looks good but missing the preview link to test!

**Comment by ElRodrigote** [comment]
> image
> on the screenshot Sushiswap appear twice, that's not supposed right?

It's not supposed to I think, but somehow we're getting 2 "Sushitrade" objects in the all platforms array. I was thinking if filtering the list by `pathcode`  or some way to identify each trade but couldn't manage to come up with a solution for that

**Comment by Diogomartf** [comment]
> > image
> > on the screenshot Sushiswap appear twice, that's not supposed right?
> 
> It's not supposed to I think, but somehow we're getting 2 "Sushitrade" objects in the all platforms array. I was thinking if filtering the list by `pathcode` or some way to identify each trade but couldn't manage to come up with a solution for that

That's odd, would be great to just have one for good UX.

**Comment by ElRodrigote** [comment]
> > > image
> > > on the screenshot Sushiswap appear twice, that's not supposed right?
> > 
> > 
> > It's not supposed to I think, but somehow we're getting 2 "Sushitrade" objects in the all platforms array. I was thinking if filtering the list by `pathcode` or some way to identify each trade but couldn't manage to come up with a solution for that
> 
> That's odd, would be great to just have one for good UX.

Seems to be working right now, apparently it was a duplicated response from the API

**Review by berteotti — APPROVED** [review]
Looks good to me!

**Pull request merged**